openshift/templates/cicdv2.yaml

```diff
@@ -0,0 +1,536 @@
+apiVersion: template.openshift.io/v1
```

Comment:
Why create a version 2 of  the cicd.yaml file?  Is this going to eventually replace cicd.yaml file once testing is done?

Reply:
I wanted a separate user to have the delete permissions. That way, if someone later sets up a script to delete everything (by mistake) using the standard user, it will fail. Basically separate responsibilities. Does that make sense?

Reply:
Okay, i'm a little confused why the only differences between cicd.yaml and cicdv2.yaml are the names and that v2 has extra 'list' permissions (no extra delete permissions).   Is that intentional?
Either way, I think it would a good idea to have some explanation in the /openshift/templates/README.md file explaining what the two different versions of the cicd files do.

Reply:
Another reason was, I didn't want to disrupt any ongoing deployments with the fix that I am trying out here. I will add the documentation in the README.md.

Reply:
Thank you